Gopkg.toml

```diff
@@ -35,5 +35,5 @@
 
 [[constraint]]
   name = "github.com/pmorie/osb-broker-lib"
-  version = "0.0.2"
+  version = "0.0.3"
```

Comment:
There are breaking changes  on 0.0.3 (originating identity) that are fixed in master. Can we cut an new release and then bump this too? Doesn't have to be in this PR, seeing this just reminded me! 😁

Reply:
I know, had that in the back of my mind as well. Wanted to get the ball rolling and get things ready, for when @pmorie cuts a release of the lib. ;) 

We can just wait for that to happen and will update this PR, as this change is not that urgent.